ansible/vars/enikshay/enikshay_public.yml

```diff
@@ -0,0 +1,287 @@
+SITE_HOST: 'enikshay.in'
+#CAS_SITE_HOST: 'cas.commcarehq.org'
+#ENIKSHAY_SITE_HOST: 'enikshay.commcarehq.org'
```

Comment:
When we migrate from india.commcarehq.org to enikshay.in, we'll want the phones to continue submitting to enikshay.commcarehq.org because changing the URL requires an app update.

We did a similar thing for icds-cas.gov.in. (the phones submit to cas.commcarehq.org)